### ISSUE TITLE
Make navigating between tracks and selection accessible  on operating systems other than Windows.

### ISSUE THREAD

**Issue by the-byte-bender** [audacity/audacity#3642]
### Your idea

Make navigating  between tracks on platforms other than Windows equally accessible by having the screenreader announce  the focused track and whether it is selected or not while navigating with the up and down arrow keys.
When you use the arrow keys on Windows, the screenreader will announce which track is now in focus and whether it is selected or not. When using the Orca screenreader on Linux, it does not announce anything when you use the arrow keys, so the only way to do things properly is to count key presses, because you know if you're on the top or bottom edge when it makes a sound when you press an arrow key. This is a major inconvenience.
Also make the selection dialog when using the left and right brackets accessible , since the screenreader on Windows will tell you whether you are manipulating minutes, seconds, or hours and what the current number is when you use the arrow keys in the dialog. This dialog  is totally inaccessible (does not speak anything other than the ok and cancel buttons ) on Linux using the orca screenreader.

### Problem to be solved

by making track navigation equally accessible  on other platforms, we will no longer have to count key strokes to determine which track we are focused on, and we will no longer have to remember whether we have selected  that track or not.
By making the selection dialog accessible  on other  platforms, we will be able to correctly set  selection markers without having to rely on pressing  brackets while the track is playing, because that is not always precise

### Additional context

I do not own a mac, so I cannot say whether these parts are already accessible or not there.
Otherwise, almost all of Audacity is usable with a screenreader, which is very fantastic, and I appreciate everyone who has contributed to accessibility.

**Comment by tbdalgaard** [comment]
Thanks for creating this issue.

I own a Mac, and can confirm that the issues are present there as well. This makes Audacity impossible to use at the moment on Mac OS. I will be happy to test any build of Audacity when this situation changes, so please keep this issue open for now, and I will follow the development from here.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
